docs/sources/installation/helm/reference.md

```diff
@@ -1993,7 +1993,7 @@ false
 		<tr>
 			<td>migrate.fromDistributed.memberlistService</td>
 			<td>string</td>
-			<td>If migrating from a distributed service, provide the distributed deployment's memberlist service DNS so the new deployment can join it's ring.</td>
+			<td>If migrating from a distributed service, provide the distributed deployment's memberlist service DNS so the new deployment can join its ring.</td>
```

Comment:
I believe that this file is generated and the one you want to edit is loki/production/helm/loki/values.yaml.

Reply:
Fixed